frontend/plugins/resources/src/components/LanguagesPage/LanguagesPage.tsx

```diff
@@ -1,12 +1,91 @@
 import React from 'react';
-import { Content, Page } from '@backstage/core';
-
-const LanguagesPage = () => (
-  <Page themeId="home">
-    <Content>
-        <div>Welcome to language page!</div>
-    </Content>
-  </Page>
-);
+import {Content, Page, Table, TableColumn, TableFilter} from '@backstage/core';
+import {DummyResponse, Edge, Resource} from "../../../../../packages/zoo-api";
+import ResourceLink from "../TableComponents/ResourceLink";
+import {Chip, Grid} from "@material-ui/core";
+import ResourceTypeLabel from "../TableComponents/ResourceTypeLabel";
+import Alert from "@material-ui/lab/Alert";
+
+
+const generateTableData: (resources: Array<Resource>) => Array<{}> = (resources) => {
+  const data: Array<{}> = [];
+  for(let i=0; i<resources.length; i++){
```

Comment:
I’d prefer:
```js
for (const resource of resources) {
```

Ref: [for...of in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)

…but I guess you could do the same with `.map()`:

```js
return resources.map(resource => ({
	usage: resource.usageCourt,
	…
}))
```

Reply:
Used map()